vocabularies/Analytics.json

```diff
@@ -161,6 +164,116 @@
+        "$Parameter": [
+          {
+            "$Name": "InputSet",
+            "$Collection": true,
+            "$Type": "Edm.EntityType",
+            "@Core.Description": "Entity set to be processed"
+          },
+          {
+            "$Name": "Levels",
+            "$Collection": true,
+            "$Type": "Analytics.MultiLevelExpandLevel",
+            "@Core.Description": "Collection of aggregation levels forming a leveled hierarchy",
+            "@Core.LongDescription": "Each element in the collection defines the properties that constitute one level.\n            A property must not be referenced by more than one level.\n            The first element in the collection defines the property names of the coarsest level,\n            the following elements define the property names of consecutively finer-grained aggregation levels.\n            The function result is the leveled hierarchy with these levels in preorder,\n            entries on the finest-grained level cannot be expanded further.\n            All referenced properties must be groupable."
+          },
+          {
+            "$Name": "Aggregation",
+            "$Collection": true,
+            "@Core.Description": "Properties to aggregate for all result entries on all levels",
+            "@Core.LongDescription": "All properties in this collection must be custom aggregates."
+          },
+          {
+            "$Name": "SiblingOrder",
+            "$Collection": true,
+            "$Type": "Analytics.MultiLevelExpandSiblingOrder",
+            "@Core.Description": "Sort specification to apply to all direct descendants of a given entry in the resulting leveled hierarchy"
+          },
+          {
+            "$Name": "ShowLevels",
+            "$Type": "Edm.Int64",
+            "@Core.Description": "Number N of levels to be shown in the initial expansion",
+            "@Core.LongDescription": "The initial expansion shows the first N levels as defined in `Levels` (0 ≤ N ≤ length of `Levels`).\n            If this parameter is omitted, all levels are shown.",
+            "@Core.OptionalParameter": {}
+          },
+          {
+            "$Name": "ExpandEntries",
+            "$Collection": true,
+            "$Type": "Analytics.MultiLevelExpandEntry",
+            "@Core.Description": "Entries with exceptional expansion",
+            "@Core.OptionalParameter": {}
+          },
+          {
+            "$Name": "SubtotalsAtBottom",
+            "$Type": "Edm.Bool",
+            "@Core.Description": "Whether to duplicate the group headers so that they appear before and after their descendants",
```

Comment:
"AtBottom" vs "before and after"

Reply:
The name reflects what you see in the UI. The description makes clear the the row appears duplicated in the result set.